lib/Map/Region/RegionProvider.ts

```diff
@@ -289,6 +342,51 @@ export default class RegionProvider {
     }
   }
 
+  loadRegionNames() {
+    if (defined(this._loadRegionNamesPromise)) {
+      return this._loadRegionNamesPromise;
+    }
+
+    const nameProp = this.nameProp || this.regionProp;
+
+    const baseuri = URI(this.analyticsWmsServer).addQuery({
+      service: "wfs",
+      version: "2.0",
+      request: "getPropertyValue",
+      typenames: this.analyticsWmsLayerName
+    });
+
+    // get the list of IDs that we will attempt to match against for this column
+
+    let url = baseuri.setQuery("valueReference", nameProp).toString();
+
+    if (this.corsProxy.shouldUseProxy(url)) {
+      url = this.corsProxy.getURL(url);
+    }
+
+    this._loadRegionNamesPromise = loadText(url).then((xml) => {
+      const obj = xml2json(xml);
+
+      if (!defined(obj.member)) {
```

Comment:
The `xml2json` function can return a string, null, and undefined (#7401), doesn't this line need to check that?